altair/utils/compiler.py

```diff
@@ -0,0 +1,11 @@
+from typing import Callable
+from altair.utils import PluginRegistry
+
+# ==============================================================================
+# Vega-Lite to Vega compiler registry
+# ==============================================================================
+VegaLiteCompilerType = Callable
```

Comment:
```suggestion
VegaLiteCompilerType = Callable[[dict], dict]
```

Let's just add some simple type hints. We'll probably expand these as part of the overall typing effort (#2951).

Reply:
Done in f043b485107ba02643755df873554978596b3482